File: JS, PHP, PYTHON/PHP/LEY OHM.php
<?php
function calcularLeyOhm() {
    echo "Calculadora de la Ley de Ohm\n";
    echo "Selecciona la variable que deseas calcular:\n";
    echo "1. Voltaje (V)\n";
    echo "2. Corriente (I)\n";
    echo "3. Resistencia (R)\n";

    $opcion = readline("Ingresa el número de la opción (1, 2 o 3): ");

    if ($opcion == '1') {
        $I = (float)readline("Ingresa la corriente (I) en amperios: ");
        $R = (float)readline("Ingresa la resistencia (R) en ohmios: ");
        $V = $I * $R;
        echo "El voltaje (V) es: $V voltios\n";

    } elseif ($opcion == '2') {
        $V = (float)readline("Ingresa el voltaje (V) en voltios: ");
        $R = (float)readline("Ingresa la resistencia (R) en ohmios: ");
        $I = $V / $R;
        echo "La corriente (I) es: $I amperios\n";

    } elseif ($opcion == '3') {
        $V = (float)readline("Ingresa el voltaje (V) en voltios: ");
        $I = (float)readline("Ingresa la corriente (I) en amperios: ");
        $R = $V / $I;
        echo "La resistencia (R) es: $R ohmios\n";

    } else {
        echo "Opción no válida. Por favor, selecciona 1, 2 o 3.\n";
    }
}

calcularLeyOhm();
?>

File: C++ y Java/C++/Ley_OHM.cpp
#include <iostream>
#include <string>
#include <iomanip>

int main()
{
  std::cout << "Calculadora de la Ley de Ohm" << std::endl;
  std::cout << "Selecciona la variable que deseas calcular:" << std::endl;
  std::cout << "1. Voltaje (V)" << std::endl;
  std::cout << "2. Corriente (I)" << std::endl;
  std::cout << "3. Resistencia (R)" << std::endl;

  std::cout << "Ingresa el numero de la opcion (1, 2 o 3): ";
  std::string opcion;
  std::getline(std::cin, opcion);

  double corrienteI, resistenciaR, voltajeV;

  if (opcion == "1")
  {
    std::cout << "Ingresa la corriente (I) en amperios: ";
    std::cin >> corrienteI;
    std::cout << "Ingresa la resistencia (R) en ohmios: ";
    std::cin >> resistenciaR;
    voltajeV = corrienteI * resistenciaR;
    std::cout << "El voltaje (V) es: " << std::fixed << std::setprecision(2) << voltajeV << " voltios" << std::endl;
  }
  else if (opcion == "2")
  {
    std::cout << "Ingresa el voltaje (V) en voltios: ";
    std::cin >> voltajeV;
    std::cout << "Ingresa la resistencia (R) en ohmios: ";
    std::cin >> resistenciaR;
    corrienteI = voltajeV / resistenciaR;
    std::cout << "La corriente (I) es: " << std::fixed << std::setprecision(2) << corrienteI << " amperios" << std::endl;
  }
  else if (opcion == "3")
  {
    std::cout << "Ingresa el voltaje (V) en voltios: ";
    std::cin >> voltajeV;
    std::cout << "Ingresa la corriente (I) en amperios: ";
    std::cin >> corrienteI;
    resistenciaR = voltajeV / corrienteI;
    std::cout << "La resistencia (R) es: " << std::fixed << std::setprecision(2) << resistenciaR << " ohmios" << std::endl;
  }
  else
  {
    std::cout << "Opcion no valida. Por favor, selecciona 1, 2 o 3." << std::endl;
  }

  return 0;
}